provider/maas/interfaces_test.go

```diff
@@ -786,9 +786,9 @@ func (s *interfacesSuite) TestMAASObjectNetworkInterfaces(c *gc.C) {
     }`, exampleInterfaceSetJSON)
 	obj := s.testMAASObject.TestServer.NewNode(nodeJSON)
 	subnetsMap := make(map[string]corenetwork.Id)
-	subnetsMap["10.250.19.0/24"] = corenetwork.Id("3")
-	subnetsMap["192.168.1.0/24"] = corenetwork.Id("0")
-	subnetsMap["192.168.20.0/24"] = corenetwork.Id("4")
+	subnetsMap["10.250.19.0/24"] = "3"
```

Comment:
Doesn't the go compiler complain about the lack of casting here?

Reply:
I realize that it's an alias to string but IIRC older Go versions still required an explicit cast

Reply:
I did the same in many places when doing the big bang space patch. All our builds have been good.

This is only in the test file too, so it's only a concern of the CI runs, which use up-to-date Go.

Reply:
What about snap builds? Aren't those built with a slightly older Go version?

Reply:
nvm; tests do not matter for snap builds...